src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -213,6 +214,7 @@ protected Object[][] getContents() {
         {"R_hostNameInCertificatePropertyDescription", "The host name to be used when validating the SQL Server Secure Sockets Layer (SSL) certificate."},
         {"R_replicationPropertyDescription", "This setting tells the server if the connection is used for replication."},
         {"R_sendTimeAsDatetimePropertyDescription", "Determines whether to use the SQL Server datetime data type to send java.sql.Time values to the database."},
+        {"R_datetimeParameterTypePropertyDescription", "Determines the datatype to use for datetime and timestamp values. Valid options are: datetime, datetime2 or datetimeoffset"},
```

Comment:
we don't usually add property value options in the error strings as there may be changed

Reply:
I was just trying to follow the pattern for what I saw in the code for the `sendTimeAsDatetime` option, as they seemed closely related.

Reply:
It looks like we haven't been consistent in the past. I'm good with including the possible values as it's helpful to the user.